plugin.py

```diff
@@ -532,7 +539,7 @@ def view(self):
         self.q_view_dlg.diffmode_chk.setCheckState(Qt.Unchecked)
         self.q_view_dlg.diffmode_chk.setEnabled(False)
 
-        pcur = versioning.Db( psycopg2.connect(self.pg_conn_info()) )
+        pcur = Db( psycopg2.connect(self.pg_conn_info()) )
```

Comment:
Same remark as previeous, I think we are using Db only twice in the `plugin.py` code